README.md

```diff
@@ -118,26 +118,26 @@ For more, see the [documentation](https://docs.astral.sh/ruff/).
 
 Ruff is available as [`ruff`](https://pypi.org/project/ruff/) on PyPI:
 
-```shell
-# With pip.
-pip install ruff
+```console
```

Comment:
Does this change how github or pypi renders the readme?

Reply:
Good point!  This does change how it appears on github (and maybe pypi too), but readme on those platforms wouldn't benefit from the patch we introduced in this PR. So, the dollar sign will still be displayed, and we can't just ignore it when copying.

Therefore, we probably shouldn't include this change in the `README.md`, right?

Reply:
Yeah. I think we should only change the markdown files in the docs folder.

Reply:
Hmm, actually it seems we only want to apply the changes to files in the `crates/` or `docs/` directories?